src/Appwrite/Utopia/Database/Validator/IndexedQueries.php

```diff
@@ -106,10 +106,8 @@ public function isValid($value): bool
             if (!$query instanceof Query) {
                 $query = Query::parse($query);
             }
-
```

Comment:
Let's keep all the blank lines here for helping with readability

Reply:
Done!